jetstream/analysis.py

```diff
@@ -516,9 +517,12 @@ def run(self, current_date: datetime, dry_run: bool = False) -> None:
 
             if time_limits is None:
                 logger.info(
-                    "Skipping %s (%s); not ready",
+                    "Skipping %s (%s); not ready [START: %s]",
                     self.config.experiment.normandy_slug,
                     period.value,
+                    self.config.experiment.start_date.strftime("%Y-%m-%d")
+                    if self.config.experiment.start_date is not None
+                    else "",
```

Comment:
We'd probably want to be explicit here that a start date hasn't been set.
```suggestion
                    else "None",
```

Reply:
Works for me, I added this logging just for debugging.